app/controllers/katello/api/v2/distributions_controller.rb

```diff
@@ -1,38 +0,0 @@
-module Katello
-  class Api::V2::DistributionsController < Api::V2::ApiController
```

Comment:
Just so I understand, @jlsherrill when you say mapping the repo object, do you mean adding the distribution attributes to the repository api (which is done here) or making a child object "distribution" of the repo object when showing a repository in the api?